### ISSUE TITLE
[doc] 3.5.2.1  Policy File Examples

### ISSUE THREAD

**Issue by bergmannf** [SUSE/doc-caasp#431]
3.5.2.1  Policy File Examples

https://susedoc.github.io/doc-caasp/master/caasp-admin/html/_security.html#_policy_file_examples

The example `psp` still shows:

```yaml
    seccomp.security.alpha.kubernetes.io/allowedProfileNames: docker/default
    seccomp.security.alpha.kubernetes.io/defaultProfileName: docker/default
```

The values generated by `skuba` now are `runtime/default` though.

**Comment by bergmannf** [comment]
Will be fixed by #430

**Comment by jordimassaguerpla** [comment]
Merged and so this bug is fixed.
